iron-flex-layout.html

```diff
@@ -259,6 +265,43 @@
       align-self: stretch;
     };
 
+    --layout-self-baseline: {
+      -ms-align-self: baseline;
+      /* currently not supported in safari */
+      -webkit-align-self: baseline;
+      align-self: baseline;
+    };
+
+    /* multi-line alignment in main axis */
+
+    /* currently not supported in ie */
+    /* currently not supported in safari */
```

Comment:
Suppress, please. If anyone else opens a similar issue, we can just link to this :)